torch/csrc/onnx/init.cpp

```diff
@@ -132,7 +132,10 @@ void initONNXBindings(PyObject* module) {
                  std::map<std::string, IValue>& params_dict,
                  int opset_version) {
                 ONNXShapeTypeInference(graph, params_dict, opset_version);
-              }))
+              }),
+          py::arg("graph"),
+          py::arg("params_dict") = true,
```

Comment:
1. Should this change be applied on main branch?
2. why `= true`?

Reply:
1. I think it's fine not doing it, because here, I just want to have a clear annotation for the usage in _fx.py. Not a bug related.
2. 😨